src/scripts/main.js

```diff
@@ -1,3 +1,80 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+const fruit = tree.firstElementChild;
+
+const redFruits = fruit.firstElementChild.firstElementChild;
```

Comment:
you shouldn't do it manually for every element (BTW top elements `Fruits` and `Fish` should also be expandable).

write a loop (or any other method you know) to do it automatically